src/simpleindex/configs.py

```diff
@@ -34,7 +35,10 @@ def __str__(self) -> str:
 @functools.lru_cache(maxsize=None)
 def _get_route_source_choices() -> typing.Dict[str, typing.Type[Route]]:
     entry_points = importlib.metadata.entry_points()
-    return {ep.name: ep.load() for ep in entry_points.get("simpleindex.routes", [])}
+    if sys.version_info < (3, 10):
+        return {ep.name: ep.load() for ep in entry_points.get("simpleindex.routes", [])}
+    else:
+        return {ep.name: ep.load() for ep in entry_points.select(group="simpleindex.routes")}
```

Comment:
```suggestion
    try:
        select = entry_points.select
    except AttributeError:
        routes = entry_points.get("simpleindex.routes", [])
    else:
        routes = select(group="simpleindex.routes")
    return {ep.name: ep.load() for ep in routes}
```

An alternative approach that avoids encoding the version number. Largely personal preference. (Also, apologies for removing `get()` - it looks like something that should've been deprecated without removing, given it appears to pretty cleanly map to another call...)

Reply:
Great suggestion, I really like it.